src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -3531,6 +3531,11 @@ public override SyntaxNode ExpressionStatement(SyntaxNode expression)
             return SyntaxFactory.ExpressionStatement((ExpressionSyntax)expression);
         }
 
+        public override SyntaxNode ConditionalAccessExpression(SyntaxNode expression, SyntaxNode whenNotNull)
+        {
+            return SyntaxFactory.ConditionalAccessExpression(Parenthesize(expression), SyntaxFactory.MemberBindingExpression((SimpleNameSyntax)whenNotNull));
```

Comment:
> The thing is: the extra factory method that is needed varies for C# (SyntaxFactory.MemberBindingExpression) and VB (SyntaxFactory.SimpleMemberAccessExpression)

I think the idea is you would do this:

```f.ConditionalAccess(left, f.MemberBindingExpression(whateverExpression))```

c# would wrap the expression in an actual MemberBindingExpression, VB would just pass the expression along untouched.